loginsightwebhookdemo/__init__.py

```diff
@@ -182,9 +182,9 @@ def sendevent(url, payload, headers=None, auth=None):
         logging.info("Headers=%s" % headers)
         logging.info("Body=%s" % payload)
         if (auth is not None):
-            r = requests.post(url, auth=auth, headers=headers, data=payload)
+            r = requests.post(url, auth=auth, headers=headers, data=payload, verify=False)
```

Comment:
I believe this will only really impact vRO as other integrations SSL certs should always be good. Can we add this directly to the vRO shim and pass it into init? I would prefer to default to security whenever possible.